src/templates/uploads.html

```diff
@@ -101,9 +101,14 @@
                 id='filecomment_{{ upload.id }}'>{{ upload.comment}}</p>
             {% endif %}
 
-            {% if mode == 'edit' and ext matches '/(json)$/i' and App.Users.userData.json_editor %}
```

Comment:
I believe there are only two modes. That's why I am not checking for one. But it might be a good idea to still keep a mode check in place for possible future modes.